LibreNMS/OS/EltexMes24xx.php

```diff
@@ -62,4 +67,28 @@ public function discoverTransceivers(): Collection
                 ]);
             });
     }
+
+    public function discoverIpv6Addresses(): Collection
+    {
+        return \SnmpQuery::allowUnordered()->enumStrings()->walk('IP-MIB::ipAddressPrefixTable')
```

Comment:
This just uses IP-MIB, does the standard discovery not work?
if this is only because of "allowUnordered"?
If that is the case, you can just set this in the OS:
```yaml
oids:
    unordered:
        - IP-MIB::ipNetToMediaPhysAddress
```

Reply:
> This just uses IP-MIB, does the standard discovery not work? 

hi @murrant 

no, it is not working
addresses are there, but prefixlen is missing
you could try with standard ipv6 module an will see
this is a reason why i using `ipAddressPrefixTable`

same as old include file which you removed in ipv6 refactor PR

Reply:
Yep, a lot of devices mess up ipAddressPrefix... this just omits it,huh? 
I wonder if that could be attempted to be handled in the base module with a manual snmpget to the ipAddressPrefixTable when ipAddressPrefix is missing or empty.

FYI, ipAddressPrefix is supposed to be a "pointer" to the row in the ipAddressPrefixTable.  Because prefix length is in the index of that table, we can get away with just parsing the prefixlen out of that pointer.